.azure-pipelines-templates/matrix.yml

```diff
@@ -32,9 +32,9 @@ parameters:
 
   test:
     NoSGX:
```

Comment:
Please make sure you don't forget to revert that change.

Reply:
reverted